src/objectid.ts

```diff
@@ -227,11 +200,7 @@ export class ObjectId {
    */
   toString(format?: BufferEncoding): string {
     // Is the id a buffer then use the buffer toString method to return the format
-    if (this.id && typeof this.id !== 'string' && 'copy' in (this.id as Buffer)) {
-      return this.id.toString(typeof format === 'string' ? format : 'hex');
-    }
-
-    return this.toHexString();
+    return this.id.toString(typeof format === 'string' ? format : 'hex');
```

Comment:
this change misses the hex string caching (not sure we care about that long term). I think for parity you want: 
```
if (format) return this.id.toString(format);
return this.toHexString();
```